DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -1,121 +1,139 @@
-            </div>
-            <div class="header-row">
-                <div class="container">
-                    <div class="col-xs-8 col-md-8 col-lg-4 logo">
-                        <h1 class="logo">
-                            @Html.ActionLink("Dancing Goat", "Index", new { controller = "Home" }, new { @class = "logo-link" })
-                        </h1>
-                    </div>
-                </div>
-            </div>
-        </header>
-        <div class="container">
-            @RenderBody()
-        </div>
-    </div>
-    <div class="footer-wrapper">
-        <footer role="contentinfo">
-            <div class="footer-container">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-md-4 col-lg-4 footer-col">
-                            <p>
-                                @Html.Action("CompanyAddress", "Home")
-                            </p>
-                        </div>
-                        <div class="col-md-4 col-lg-4 footer-col">
-                            <h3>@Resources.DancingGoat.FollowUs_Title</h3>
-                            <a class="followus-link" href="https://www.facebook.com/Dancing.Goat" target="_blank">
-                                <img alt="@Resources.DancingGoat.FollowUsOnFacebook" class="" src="~/Content/Images/facebook-icon.png" title="@Resources.DancingGoat.FollowUsOnFacebook">
-                            </a>
-                            <a class="followus-link" href="https://twitter.com/DancingGoat78" target="_blank">
-                                <img alt="@Resources.DancingGoat.FollowUsOnTwitter" class="" src="~/Content/Images/twitter-icon.png" title="@Resources.DancingGoat.FollowUsOnTwitter">
-                            </a>
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div class="container copyright">
-                Copyright &copy; @DateTime.Now.Year Dancing Goat. All rights reserved.
-            </div>
-        </footer>
-    </div>
-
-    @Scripts.Render("~/bundles/jquery")
-    @Scripts.Render("~/bundles/jqueryval")
-    @RenderSection("scripts", required: false)
-
-    <script type="text/javascript">
-        !function(){var a='https://engage-ket.kenticocloud.com/js',b=document,c=b.createElement('script'),d=b.getElementsByTagName('script')[0];c.type='text/javascript',c.async=!0,c.defer=!0,c.src=a+'?d='+document.domain,d.parentNode.insertBefore(c,d)}(),window.ket=window.ket||function(){(ket.q=ket.q||[]).push(arguments)};
-        ket('start', '@ConfigurationManager.AppSettings["ProjectId"]');
-    </script>
-    <script type="text/javascript">
-        function toggleEditMode(isChecked) {
-            const body = document.getElementsByTagName('body')[0];
-            isChecked ? body.classList.add('edit-mode') : body.classList.remove('edit-mode');
-        }
-    </script>
-
-
-</body>
+﻿@using DancingGoat.Helpers

```

Comment:
Formatting is still changes, and may introduce conflicts, please reset this file and make only necessary change